src/coreclr/nativeaot/Runtime/startup.cpp

```diff
@@ -372,7 +372,12 @@ extern "C" bool RhInitialize(bool isDll)
 #endif
 
 #if defined(HOST_WINDOWS) || defined(FEATURE_PERFTRACING)
-    atexit(&OnProcessExit);
+#if defined(HOST_WINDOWS)
+    at_quick_exit
```

Comment:
This is changing the behavior of shipping release bits. `at_quick_exit` handlers are not called on normal program termination. 

`OnProcessExit` is no longer called on normal program termination -> the tracing shutdown is no longer during shutdown -> we should see tracing tests fail -> we have a test hole since no tracing tests are failing.

I am leaving a suggestion with a fix.

Reply:
Filed #114461 for the test hole.